src/maps/views.py

```diff
@@ -52,3 +66,35 @@ def get(self, *args, **kwargs):
 
         # all good, return the response
         return response
+
+    def is_endpoint_valid(self, path: str) -> None:
+        """Validate request path against whitelisted endpoints or raise PermDenied
+        """
+        endpoint = path.replace(self.PROXY_URL, "", 1)
+        if endpoint not in self.VALID_ENDPOINTS:
+            logger.warning("Maps endpoint was invalid: '%s' valid endpoints: %s",
+                           endpoint, self.VALID_ENDPOINTS)
+            raise PermissionDenied("No thanks")
+
+    def sanitize_path(self, path: str) -> str:
+        """Sanitize path by removing PROXY_URL and set 'transparent' value to upper
+        """
+        new_path = path.replace(self.PROXY_URL, "", 1)
+        sanitized_path = re.sub(
+            r"(transparent=)(true|false)",
```

Comment:
what does transparent do?

Reply:
To be honest.. No idea. But during my tests it was a required parameter for calling the API, and it wants the "true/false" value to be uppercase.

Reply:
:+1: